Ladybird/AppKit/UI/Inspector.h

```diff
@@ -21,7 +21,7 @@
 
 - (void)selectHoveredElement;
 
-- (void)setShowCloseButton:(BOOL)is_visible;
+- (void)setIsWindowed:(BOOL)is_windowed;
```

Comment:
This commit just renames code you added in an earlier commit. Please squash these changes into that.